### DISCUSSION TITLE
Adding new technology

### DISCUSSION THREAD

**Discussion by joy4kim** [JGCRI/gcam-core#300] [Modifying GCAM]
I'm trying to figure out how I can add new technology (fuel) option for aviation sector. The technology is named as sustainable aviation fuel. Currently, the fuel of air domestic and international is represented by a "liquid", according to gcamdata input, A54.globaltranTech_shrwt_revised.csv. If I want to add new biofuel for air transportation, where can I start with? Can I add a row below the "liquid" fuel for the air international? My ultimate goal is to look at emission reduction impacts of adding it. 
If you know any other tutorial or documentation that I can get an idea of how to add new technology, please let me know. Thanks!

**Comment by PaulWolfram** [comment]
Hi Joyce, 

yes adding a new line to A54.globaltranTech_shrwt_revised.csv is a good start, however there may be more transport-related files where you will have to add a row in a similar fashion. Make sure you identify all of them. If you're a Windows user, the GitBash command `grep -r your_search_term` can help you identify all the files that you need to modify. Be sure to remove any trailing commas in any of the .csv files that you modified using a text editor. 

Best, Paul
